mypy.ini

```diff
@@ -5,7 +5,7 @@ ignore_missing_imports = True
 strict_equality = True
 warn_redundant_casts = True
 warn_unused_configs = True
-warn_unused_ignores = True
+warn_unused_ignores = False
```

Comment:
Changed this as it seems that mypy messages are incorrect (the # type: ignore is needed, but mypy claims it is not)

Reply:
Just an aside, but I've found this usually to happen when some imports are not quite right in some way